src/Medology/Behat/Traits/GathersContexts.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Medology\Behat\Traits;
+
+use Behat\Behat\Context\Environment\InitializedContextEnvironment;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+use RuntimeException;
+
+/**
+ * Interface for Behat contexts that utilize other Contexts.
+ *
+ * This interface defines a single function that is triggered before each Behat scenario. The implementing class
+ * should use this function to gather any contexts from the environment that it needs to interact with.
+ *
+ * @see http://behat-docs.readthedocs.io/en/mvp1.0/cookbooks/gathering_contexts_when_using_multiple_contexts.html
+ */
+trait GathersContexts
+{
+    /**
+     * Gets the environment in order to do gathersContexts.
+     *
+     * This should be called at the beginning of any implementation of gathersContexts, and will return the environment.
+     *
+     * @throws RuntimeException if the current environment is not initialized
+     */
+    public function getEnvironment(BeforeScenarioScope $scope): InitializedContextEnvironment
+    {
```

Comment:
There are a few instances of the opening curly bracket being on the wrong line. Is the style fixer doing this? Maybe the style fixer isn't working on this project.

```suggestion
    public function getEnvironment(BeforeScenarioScope $scope): InitializedContextEnvironment {
```